UnitsNet/UnitDefinitions/Acceleration.json

```diff
@@ -1,20 +1,56 @@
 ﻿{
-    "Name": "Acceleration",
-    "BaseUnit": "MeterPerSecondSquared",
-    "XmlDoc": "Acceleration, in physics, is the rate at which the velocity of an object changes over time. An object's acceleration is the net result of any and all forces acting on the object, as described by Newton's Second Law. The SI unit for acceleration is the Meter per second squared (m/s²). Accelerations are vector quantities (they have magnitude and direction) and add according to the parallelogram law. As a vector, the calculated net force is equal to the product of the object's mass (a scalar quantity) and the acceleration.",
-    "Units": [
+  "Name": "Acceleration",
+  "BaseUnit": "MeterPerSecondSquared",
+  "XmlDoc": "Acceleration, in physics, is the rate at which the velocity of an object changes over time. An object's acceleration is the net result of any and all forces acting on the object, as described by Newton's Second Law. The SI unit for acceleration is the Meter per second squared (m/s²). Accelerations are vector quantities (they have magnitude and direction) and add according to the parallelogram law. As a vector, the calculated net force is equal to the product of the object's mass (a scalar quantity) and the acceleration.",
+  "Units": [
+    {
+      "SingularName": "MeterPerSecondSquared",
+      "PluralName": "MeterPerSecondSquared",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Prefixes": [ "Nano", "Micro", "Milli", "Centi", "Deci", "Kilo" ],
+      "Localization": [
         {
-            "SingularName": "MeterPerSecondSquared",
-            "PluralName": "MeterPerSecondSquared",
-            "FromUnitToBaseFunc": "x",
-            "FromBaseToUnitFunc": "x",
-            "Prefixes": ["Nano","Micro","Milli","Centi","Deci","Kilo"],
-            "Localization": [
-                {
-                    "Culture": "en-US",
-                    "Abbreviations": ["m/s²"]
-                }
-            ]
-        }        
-    ]
+          "Culture": "en-US",
+          "Abbreviations": [ "m/s²" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "InchPerSecondSquared",
+      "PluralName": "InchesPerSecondSquared",
+      "FromUnitToBaseFunc": "x/39.3700787",
+      "FromBaseToUnitFunc": "x*39.3700787",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "in/s²" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "FootPerSecondSquared",
+      "PluralName": "FeetPerSecondSquared",
+      "FromUnitToBaseFunc": "x*0.304800",
+      "FromBaseToUnitFunc": "x*3.28084",
```

Comment:
Use `x/0.304800` to preserve precision.

Reply:
Should we also include abbreviations that do not rely on the superscript?  ft/s^2

Reply:
On second thought, I think I would say no, because we haven't usually done so. But I get your point and as in my previous comment, I am thinking maybe the library should generally allow parsing these ^2 variants as substitutes for superscripts.

Reply:
Fair enough; thanks.